profiles/api/goals.php

```diff
@@ -1,24 +1,91 @@
 <?php
 require_once($_SERVER['DOCUMENT_ROOT'] . "/global/php/functions.php");
+require_once($_SERVER['DOCUMENT_ROOT'] . "/global/php/api_controller.php");

```

Comment:
would prefer this to be a function in functions.php which includes it. easier if we move the file in the future, which will happen since the /global/php folder is a mess

Reply:
i.e.
```php
function import_api_controller() {
    require_once($_SERVER['DOCUMENT_ROOT'] . "/global/php/api_controller.php");
}
```
?

Reply:
yep, maybe without the import_ for time being. we have other things which import files in the same way

Reply:
^ * definitely want to get an actually good naming convention for these functions at some point but for now best to just follow what we have